### ISSUE TITLE
hey 

### ISSUE THREAD

**Issue by CTPCORP** [nintendo-mod-1/Nintendo-GBU-Software#2]
Dude is the Python the touchscreen? Thanks

**Comment by pokemasterblaze** [comment]
No we need to code it so that it's touch-screen

On September 1, 2015, at 8:50 PM, CTPCORP notifications@github.com wrote:

Dude is the Python the touchscreen? Thanks

**Comment by CTPCORP** [comment]
Oooooh ok Got it. Should we start a Kickstart after we figure out how to
mass produce.

**Comment by pokemasterblaze** [comment]
Yeah, but if we do that we first have to get it through with Nintendo (so we can use their games ) or change the name and games. I'd prefer the first one.

On September 2, 2015, at 7:19 AM, CTPCORP notifications@github.com wrote:

Oooooh ok Got it. Should we start a Kickstart after we figure out how to
mass produce.

**Comment by CTPCORP** [comment]
hay dude give me ur # so I can call you.

**Comment by pokemasterblaze** [comment]
My phone got turned off. I lost it too much but I can call you on Google hangouts.

On September 13, 2015, at 12:44 PM, CTPCORP notifications@github.com wrote:

hay dude give me ur # so I can call you.

**Comment by CTPCORP** [comment]
Dude wats ur gmail?

**Comment by CTPCORP** [comment]
???
